deploy.sh

```diff
@@ -228,22 +228,15 @@ function init_terraform {
 # Check if SSH key-pair is provided else use users key or create a new one
 #-------------------------------------------------------------------------
 function verify_data {
-  if [ -s "./pull-secret.txt" ]; then
-    log "Found pull-secret.txt in current directory"
-    cp pull-secret.txt ./"$ARTIFACTS_DIR"/data/
+  if [ -s "./"$ARTIFACTS_DIR"/data/pull-secret.txt" ]; then
```

Comment:
User may not know what is data directory or its path.
If not found in data dir then check current dir.. if found copy it to data as before, else error out.